libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper.stories.ts

```diff
@@ -0,0 +1,162 @@
+  decorators: [
+    moduleMetadata({
+      imports: [
+        CommonModule,
+        ReactiveFormsModule,
+        SdsFormlyModule,
+        FormsModule,
+        FormlyModule.forRoot(),
+        FormlyStepperIntroductionModule,
+        NoopAnimationsModule,
+        FormlyStepperBasicModule,
+        FormlyStepperIntroductionModule,
+        RouterTestingModule,
+        FormlyStepperAdvancedModule,
+        UswdsStepperModule,
+      ],
+      providers: [SdsStepper],
+    }),
+  ],
+  argTypes: {
+    label: templateOptions,
+    placeholder: templateOptions,
+    description: templateOptions,
+    required: templateOptions,
+    disabled: templateOptions,
+    tooltipText: templateOptions,
+  },
+} as Meta;
+
+export const Introduction: Story = (args) => ({
+  template: '<sds-formly-stepper-introduction></sds-formly-stepper-introduction>',
+  props: args,
+});
+Introduction.parameters = { options: { showPanel: false } };
+
+export const Uneven: Story = (args) => ({
+  template: '<sds-formly-stepper-basic></sds-formly-stepper-basic>',
+  props: args,
+});
+Uneven.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
+    'storybook/formly/formly-stepper/formly-stepper-basic',
+    'FormlyStepperBasicModule',
+    'sds-formly-stepper-basic',
+    [
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-basic/formly-stepper-basic-uneven.component.ts',
+        'ts',
+        false
+      ),
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-basic/formly-stepper-basic-uneven.component.html',
+        'html',
+        false
+      ),
+    ]
+  ),
+  stackblitzLink: generateStackblitzLink('formly-stepper', 'basic'),
+};
+
+export const USWDS: Story = (args) => ({
+  template: '<gsa-sam-uswds-stepper></gsa-sam-uswds-stepper>',
+  props: args,
+});
+USWDS.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
+    'storybook/formly/formly-stepper/formly-stepper-uswds',
+    'UswdsStepperModule',
+    'gsa-sam-uswds-stepper',
+    [
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-uswds/formly-stepper-uswds-custom.component.ts',
+        'ts',
+        false
+      ),
+      createCodePreviewTabData(
+        'storybook/formly/formly-stepper/formly-stepper-uswds/formly-stepper-uswds-custom.component.html',
+        'html',
+        false
+      ),
+    ]
+  ),
+  stackblitzLink: generateStackblitzLink('formly-stepper', 'uswds'),
+};
+
+export const Advanced: Story = (args) => ({
+  template: '<stepper-advanced-demo></stepper-advanced-demo>',
+  props: args,
+});
+Advanced.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
```

Comment:
Stackblitz is failing to initialize:
Error in src/app/formly-stepper-advanced.component.html (49:57)
Type '{ key: string; templateOptions: { placeholder: string; }; fieldGroup: ({ key: string; fieldGroupClassName: string; fieldGroup: ({ className: string; key: string; type: string; templateOptions: { ...; }; } | { ...; })[]; className?: undefined; type?: undefined; templateOptions?: undefined; modelOptions?: undefined; }...' is not assignable to type 'FormlyFieldConfig'.
Types of property 'fieldGroup' are incompatible.
Type '({ key: string; fieldGroupClassName: string; fieldGroup: ({ className: string; key: string; type: string; templateOptions: { label: string; hideOptional: boolean; options: { label: string; value: string; }[]; required: boolean; showError?: undefined; }; } | { ...; })[]; className?: undefined; type?: undefined; templ...' is not assignable to type 'FormlyFieldConfig[]'.
Type '{ key: string; fieldGroupClassName: string; fieldGroup: ({ className: string; key: string; type: string; templateOptions: { label: string; hideOptional: boolean; options: { label: string; value: string; }[]; required: boolean; showError?: undefined; }; } | { ...; })[]; className?: undefined; type?: undefined; templa...' is not assignable to type 'FormlyFieldConfig'.
Type '{ className: string; type: string; key: string; templateOptions: { label: string; required: boolean; showError: boolean; }; modelOptions: { updateOn: string; }; fieldGroupClassName?: undefined; fieldGroup?: undefined; }' is not assignable to type 'FormlyFieldConfig'.
The types of 'modelOptions.updateOn' are incompatible between these types.
Type 'string' is not assignable to type '"blur" | "change" | "submit"'.